From 564d4c09263c021aa7e651edbfbd22e8b895a3bf Mon Sep 17 00:00:00 2001
From: Jozef Kenyeres <jkenyeres@sipwise.com>
Date: Thu, 29 Dec 2022 10:14:18 +0100
Subject: MT#55831 fix failed relay of 200OK without SDP

fix failed relay of 200OK without SDP
(real ticket number: TT#9521)

Index: sems/core/AmOfferAnswer.cpp
===================================================================
--- sems.orig/core/AmOfferAnswer.cpp
+++ sems/core/AmOfferAnswer.cpp
@@ -392,6 +392,9 @@ int AmOfferAnswer::onReplyOut(AmSipReply
       if (reply.code == 183 && reply.cseq_method == SIP_METH_INVITE) {
         // just ignore if no SDP is generated (required for B2B)
       }
+      else if (reply.code == 200 && reply.cseq_method == SIP_METH_INVITE && state == OA_Completed) {
+        // just ignore if no SDP is generated (required for B2B)
+      }
       else return -1;
     }
     else {
